docs/workflows/user-provisioning.md

```diff
@@ -153,3 +153,9 @@ func main() {
 
 </TabItem>
 </Tabs>
+
+### Groups
+
+You can also create groups and add or remove identities from groups in the Beyond Identity API.
+
+For deleting, updating, and managing groups via API, see the [API documentation](https://developer.beyondidentity.com/api/v1#tag/Groups).
```

Comment:
Is this duplicated from the section above?

Reply:
There's a section above to the console, but this is new for the API to separate that weird sentence that was joined with creating an identity